Security/Authentication/Provider/JWTProvider.php

```diff
@@ -44,22 +44,30 @@ class JWTProvider implements AuthenticationProviderInterface
      */
     protected $userIdentityField;
 
+    /**
+     * @var string
+     */
+    protected $userIdClaim;
```

Comment:
let's make it private to reduce the maintenance cost, it's exposed through a getter